app/Http/Controllers/UsersController.php

```diff
@@ -102,19 +102,15 @@ public function postCreate(SaveUserRequest $request)
     {
 
         $user = new User;
-        $user->first_name  = $data['first_name']= e($request->input('first_name'));
-        $user->last_name = e($request->input('last_name'));
+        //Username, email, and password need to be handled specially because the need to respect config values on an edit.
```

Comment:
The comment may have been worded poorly, basically it was saying that because postEdit needs to do some checks and things on those items, they can't be handled in the central method and needed to be handled directly in postCreate.